lib/schema.json

```diff
@@ -19,6 +19,13 @@
     "short": "o",
     "value": "[path]"
   },
+  {
+    "long": "replace",
+    "description": "fix and replace file(s)",
+    "short": "p",
```

Comment:
I'm not a short arguments kind of guy, so if anyone has a better idea for the short just shout 👍